kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -0,0 +1,27 @@
+# Main deployment for API server
+apiVersion: apps/v1beta1
+kind: Deployment
```

Comment:
So where will the dump generating instance be ? The one that consolidates everything ?